client/my-sites/checkout/src/components/payment-methods-filter.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+	Button,
+	__experimentalSpacer as Spacer,
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+export interface PaymentMethodFilterProps {
+	areStoredCardsFiltered: boolean | undefined;
+	isBusinessCardsFilterEmpty: boolean | undefined;
+}
+
+export const PaymentMethodFilterNotice = ( {
+	isBusinessCardsFilterEmpty,
+}: {
+	isBusinessCardsFilterEmpty?: boolean;
+} ) => {
+	let noticeText;
+
+	if ( isBusinessCardsFilterEmpty ) {
+		noticeText = __( 'There are no stored business cards on your account. Please add one below.' );
+	} else {
+		noticeText = __( 'Payment methods filtered by Business Cards' );
+	}
+
+	return (
+		<Text color="var(--studio-gray-60)" size="14px">
+			{ noticeText }
+		</Text>
+	);
+};
+
+export const PaymentMethodFilterButton = () => {
+	// TODO - Implement useMakeActiveStep once https://github.com/Automattic/wp-calypso/pull/101753 is merged
```

Comment:
This will be swapped out after the useMakeStepActive hook is merged https://github.com/Automattic/wp-calypso/pull/101753

Reply:
It's merged now! Thanks!